src/compiler/scala/tools/nsc/backend/jvm/BTypes.scala

```diff
@@ -1145,7 +1145,8 @@ object BTypes {
         i += 1
       }
       scala.util.Sorting.quickSort(result)(Ordering.by(_._1))
-      result
+      import scala.collection.immutable.ArraySeq
+      ArraySeq.unsafeWrapArray(result)
```

Comment:
Is the style local imports or FQNs? Pace that one "DOES NOT WORK in general" commit, `collection.immutable.ArraySeq` is a nice middle ground. Even more so if you `import collection._`, because who doesn't need a nice collection?

Reply:
FTR I prefer local import. On this scale of linting, I tried to avoid messing with imports in large scopes except for removing unused.

Reply:
I imported the unsafe method at top. It looks much cleaner.